packages-next/types/src/config/access-control.ts

```diff
@@ -5,50 +5,40 @@ type BaseAccessArgs = {
   session: any;
   listKey: string;
   context: KeystoneContext;
-  // idk if this is optional or not
-  gqlName?: string;
 };
 
 type CreateAccessArgs<GeneratedListTypes extends BaseGeneratedListTypes> = BaseAccessArgs & {
   operation: 'create';
   /**
    * The input passed in from the GraphQL API
    */
-  originalInput?:
-    | GeneratedListTypes['inputs']['create']
-    | readonly { readonly data: GeneratedListTypes['inputs']['create'] }[];
+  originalInput: GeneratedListTypes['inputs']['create'];
 };
 
-type CreateAccessControl<GeneratedListTypes extends BaseGeneratedListTypes> =
+export type CreateAccessControl<GeneratedListTypes extends BaseGeneratedListTypes> =
   | boolean
   | ((args: CreateAccessArgs<GeneratedListTypes>) => MaybePromise<boolean>);
 
 type ReadAccessArgs = BaseAccessArgs & { operation: 'read' };
 
-type ReadListAccessControl<GeneratedListTypes extends BaseGeneratedListTypes> =
+export type ReadListAccessControl<GeneratedListTypes extends BaseGeneratedListTypes> =
   | boolean
   | GeneratedListTypes['inputs']['where']
   | ((args: ReadAccessArgs) => MaybePromise<GeneratedListTypes['inputs']['where'] | boolean>);
 
 type UpdateAccessArgs<GeneratedListTypes extends BaseGeneratedListTypes> = BaseAccessArgs & {
   /**
-   * The id being updated if a single item is being updated
+   * The id being updated
    */
-  itemId?: string;
-  /**
-   * The ids being updated if many items are being updated
-   */
-  itemIds?: string[];
+  itemId: string;
```

Comment:
References to `itemIds` in `apis/access-control.mdx` need to be removed to match this change.